apps/microtvm/zephyr/template_project/CMakeLists.txt.template

```diff
@@ -32,11 +32,16 @@ if(${ENABLE_CMSIS})
   set(CMSIS_PATH <CMSIS_PATH>)
 
   file(GLOB_RECURSE cmsis_lib_srcs
-    ${CMSIS_PATH}/CMSIS/NN/Source/SoftmaxFunctions/*.c
+    ${CMSIS_PATH}/CMSIS/NN/Source/ActivationFunctions/*.c]
+    ${CMSIS_PATH}/CMSIS/NN/Source/BasicMathFunctions/*.c
+    ${CMSIS_PATH}/CMSIS/NN/Source/ConcatenationFunctions/*.c
     ${CMSIS_PATH}/CMSIS/NN/Source/ConvolutionFunctions/*.c
     ${CMSIS_PATH}/CMSIS/NN/Source/FullyConnectedFunctions/*.c
     ${CMSIS_PATH}/CMSIS/NN/Source/NNSupportFunctions/*.c
     ${CMSIS_PATH}/CMSIS/NN/Source/PoolingFunctions/*.c
+    ${CMSIS_PATH}/CMSIS/NN/Source/ReshapeFunctions/*.c
+    ${CMSIS_PATH}/CMSIS/NN/Source/SVDFunctions/*.c
```

Comment:
Interesting to see `SVDF` here. Where it is used?

Reply:
Given SVDF is not supported in the stack, I suggest removing it.

Reply:
done, thanks!